### PR TITLE
Bump up Twilio version to 5.0.0

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

File: SMS2FA.Web/App_Start/IdentityConfig.cs [hunk 1]
```diff
@@ -27,37 +30,40 @@ public class SmsService : IIdentityMessageService
...
-        public SmsService(TwilioRestClient client)
+        public SmsService(ITwilioRestClient client)
         {
-            _client = client;
+            TwilioClient.Init(Config.AccountSid, Config.AuthToken);
+            TwilioClient.SetRestClient(client);
         }
 
-        public Task SendAsync(IdentityMessage message)
+        public async Task SendAsync(IdentityMessage message)
```
Comment (joliveros) [inline SMS2FA.Web/App_Start/IdentityConfig.cs]: this should return a `Task<ResourceSet<MessageResource>>`, no?

File: SMS2FA.Web/App_Start/IdentityConfig.cs [hunk 2]
```diff
@@ -106,7 +113,8 @@ public static ApplicationUserManager Create(IdentityFactoryOptions<ApplicationUs
     // Configure the application sign-in manager which is used in this application.
     public class ApplicationSignInManager : SignInManager<ApplicationUser, string>
     {
-        public ApplicationSignInManager(ApplicationUserManager userManager, IAuthenticationManager authenticationManager)
+        public ApplicationSignInManager(ApplicationUserManager userManager,
```
Comment (joliveros) [inline SMS2FA.Web/App_Start/IdentityConfig.cs:111]: +1 on wrapping those lines!

File: SMS2FA.Web/Controllers/AccountController.cs [hunk 3]
```diff
@@ -72,15 +77,19 @@ public ActionResult Login(string returnUrl)
 
             // This doesn't count login failures towards account lockout
             // To enable password failures to trigger account lockout, change to shouldLockout: true
-            var result = await SignInManager.PasswordSignInAsync(model.Email, model.Password, model.RememberMe, shouldLockout: false);
+            var result = await SignInManager.PasswordSignInAsync(model.Email,
```
Comment (joliveros) [inline SMS2FA.Web/Controllers/AccountController.cs:80]: Would this look better this way? What do you think @acamino, @hortega ?

```csharp
var result = await SignInManager.PasswordSignInAsync(
    model.Email,
    model.Password,
    model.RememberMe,
    shouldLockout: false
);
```

File: SMS2FA.Web/Controllers/AccountController.cs [hunk 4]
```diff
@@ -113,11 +126,15 @@ public ActionResult Login(string returnUrl)
...
+            // The following code protects for brute force attacks against the two factor codes.
+            // If a user enters incorrect codes for a specified amount of time then the user account
+            // will be locked out for a specified amount of time.
             // You can configure the account lockout settings in IdentityConfig
-            var result = await SignInManager.TwoFactorSignInAsync(model.Provider, model.Code, isPersistent:  model.RememberMe, rememberBrowser: model.RememberBrowser);
+            var result = await SignInManager.TwoFactorSignInAsync(
+                model.Provider,
+                model.Code,
+                isPersistent:  model.RememberMe,
+                rememberBrowser: model.RememberBrowser);
```
Comment (joliveros) [inline SMS2FA.Web/Controllers/AccountController.cs:137]: I would add some spacing here to break up the blocks of code for readability

File: SMS2FA.Web/SMS2FA.Web.csproj [hunk 5]
```diff
@@ -48,13 +48,21 @@
...
+    </Reference>
     <Reference Include="Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.1\lib\net45\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.9.0.2-beta1\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (joliveros) [inline SMS2FA.Web/SMS2FA.Web.csproj]: perhaps you could use Newtonsoft.Json v9.0.1 instead?

**Review by joliveros — COMMENTED** [review]
@hortega, please check minor comments. Everything else looks great!

File: SMS2FA.Web/App_Start/IdentityConfig.cs (hunk 1 above)
Comment (hortega) [reply SMS2FA.Web/App_Start/IdentityConfig.cs]: `SMSService` implements `IIdentityMessageService` which is a builtin interface, hence we can't change the return type. It needs to implement it as later the service is used inside a UserManager.
In any case we don't have use for the returned ResourceSet in this code.

File: SMS2FA.Web/Controllers/AccountController.cs (hunk 3 above)
Comment (hortega) [reply SMS2FA.Web/Controllers/AccountController.cs:80]: I think it's easier to read as it is. Let's look for third opinion

**Comment by joliveros** [comment]
LGTM @hortega,

nice work!

**Comment by hortega** [comment]
thanks for the review @joliveros

**Pull request merged**